api/app/controllers/opennode.controller.ts

```diff
@@ -0,0 +1,64 @@
+require("dotenv").config();
+import { Request, Response } from "express";
+const opennode = require("opennode");
+opennode.setCredentials(
+  process.env.OPENNODE_API_KEY,
+  process.env.OPENNODE_API_ENV
+);
+import { WithdrawalResponse,OpenNodeResponse } from "../types/lightning";
+
+export async function lninvoiceWithdrawal(req: Request, res: Response) {
+  const { invoice } = req.body;
+
+  if (invoice === "") {
+    return res.status(400).json({ message: "Please enter a valid invoice" });
```

Comment:
A better error message would be `invoice cannot be empty`. We would also want to validate the invoice if its stale or alrwady paid